apps/bfd-model/bfd-model-rif-samples/src/main/java/gov/cms/bfd/model/rif/samples/StaticRifResource.java

```diff
@@ -323,8 +325,39 @@ public RifFileType getRifFileType() {
     return rifFileType;
   }
 
-  /** @return the number of beneficiaries/claims/drug events in the RIF file */
+  /** @return the number of beneficiaries/claims/drug events in the RIF file excluding line items */
   public int getRecordCount() {
+    if (recordCount == -1) {
+      synchronized (this) {
```

Comment:
is there a threading concern here? I think this could also go away if we just compute record count up front in the constructor.

Reply:
Done